src/templates/api-endpoint.njk

```diff
@@ -19,6 +19,7 @@
  */
 
 import { GoogleApis } from '../..';
+import {OAuth2Client, JWT, Compute, UserRefreshClient} from 'google-auth-library';
```

Comment:
nit: this line and its neighbors are formatted differently: `{ GoogleApis }` but `{AxiosPromise}`